app/uk/gov/hmrc/lisaapi/controllers/APIVersioning.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.lisaapi.controllers
+
+import play.api.Logger
+import play.api.http.HeaderNames.ACCEPT
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+trait APIVersioning {
```

Comment:
@chriswaugh Please could you confirm you're happy with this file?

Reply:
Yup LGTM :+1: